Sources/KSCrashRecording/KSCrashConfiguration.m

```diff
@@ -35,18 +35,35 @@ - (instancetype)init
     self = [super init];
     if (self)
     {
-        _monitors = KSCrashMonitorTypeProductionSafeMinimal;
-        _userInfoJSON = nil;
-        _deadlockWatchdogInterval = 0.0;
-        _enableQueueNameSearch = NO;
-        _enableMemoryIntrospection = NO;
+        KSCrashCConfiguration cConfig = KSCrashCConfiguration_Default();
+        _monitors = cConfig.monitors;
+
+        if (cConfig.userInfoJSON != NULL)
+        {
+            NSData *data = [NSData dataWithBytes:cConfig.userInfoJSON length:strlen(cConfig.userInfoJSON)];
+            NSError *error = nil;
+            _userInfoJSON = [NSJSONSerialization JSONObjectWithData:data options:0 error:&error];
+            if (error)
+            {
+                _userInfoJSON = nil;  // Handle the error appropriately
+            }
+        }
+        else
+        {
+            _userInfoJSON = nil;
+        }
+
+        _deadlockWatchdogInterval = cConfig.deadlockWatchdogInterval;
+        _enableQueueNameSearch = cConfig.enableQueueNameSearch ? YES : NO;
```

Comment:
Is there a reason for ternary operator here and below?
```suggestion
        _enableQueueNameSearch = (BOOL)cConfig.enableQueueNameSearch;
```

Reply:
Because `YES` is `1`, but `true` is `!1`. I think cast wouldn't fix it.